### DISCUSSION TITLE
User subscription management with new pricing plan

### DISCUSSION THREAD

**Discussion by RedBlob** [Budibase/budibase#10764] [Help] [answered]
Hello,

If I have bought a Premium plan with 5 users and I have paid for the annual plan for all 5 users, if a user renouce after 3 months to use BB can I associate the remaining 9 months to a new user with a new email ?

Or once a user unsubscribes from my service the remaining months of subscription are lost and by introducing a new email a new subscription starts again for 12 month ?

Thank You
Best Regards

**Comment by ConorWebb96** [comment, marked as answer]
Hey @RedBlob,

You can reuse the license for new users or email addresses - we take the current end-user count, not the historical total. Note: you’ll have to delete users from your bb instance if they’re no longer using it, otherwise they will still be counted. 

If you increase your end-user count, we'll calculate a prorated amount for the extra time. This payment will be charged the following month using the credit card you provided. 

I hope the above answers your question.
If you have more questions, let us know!

**Reply by Rory-Powell** [reply]
To add to Conors point on prorations, the following behaviour is observed:

- User added = Prorated charge for the remaining time in the current cycle (annual or monthly)
- User removed = Prorated invoice credit for the remaining time in the current cycle (annual or monthly)

Monthly Plans
- Invoice generated at the end of the monthly billing cycle will include any proration items

Annual Plans
- Proration invoices on annual plans are generated on the 1st of every month (or the next working day)
- Proration invoices on annual plans for May 2023 will be generated on 5th June 2023

**Comment by RedBlob** [comment]
Hi @Rory-Powell and @ConorWebb96 ,

Thank You so much for your explanation.

Best Regards
